config/install/rest.settings.yml

```diff
@@ -0,0 +1 @@
+bc_entity_resource_permissions: false
```

Comment:
I get an error from this config when installing the module:
`Configuration objects (rest.settings, serialization.settings) provided by dennis_term_manager already exist in active configuration`